src/core/KuzzleEventEmitter.ts

```diff
@@ -1,8 +1,10 @@
+import { DisconnectionOrigin } from "src/protocols/DisconnectionOrigin";
```

Comment:
This import should be relative otherwise the build will fail

Reply:
Damn, I'm getting caught every time by auto-import.